### PR TITLE
Labelling interface

### PR DESCRIPTION
**Pull request by ramhound**
Labelling UI in place for both live labelling and for historical labelling through the graph

### CODE REVIEW

File: seadssite/static/js/graph.js [hunk 1]
```diff
@@ -21,20 +86,23 @@ var repeater = null;
 //repeat in ms
 function pick(func, repeat) {
     if (repeater) {
-	clearInterval(repeater);
```
Comment (iangudger) [inline seadssite/static/js/graph.js:24]: Probably better not to mix tabs and spaces for indentation.

File: seadssite/static/js/graph.js [hunk 2]
```diff
@@ -77,53 +150,179 @@ function fetch_graph(url) {
...
+    });
+
+    pick(pick_live, 10 * 1000);
+
+    pie();
+
+    bar();
+
+    //generating url for weekly energy data here for now as test
+    //var url = "http://db.sead.systems:8080/466419818?start_time=" + start + "&end_time=" + end + "&list_format=energy&type=P&device=Panel1&granularity=86400";
```
Comment (iangudger) [inline seadssite/static/js/graph.js]: Remove commented out code.

File: seadssite/static/js/graph.js [hunk 3]
```diff
@@ -2,18 +2,83 @@
 
 var c3 = require('c3');
 
-var HOUR_SECONDS = 60*60;
-var DAY_SECONDS = HOUR_SECONDS*24;
+var HOUR_SECONDS = 60 * 60;
+var DAY_SECONDS = HOUR_SECONDS * 24;
 
 //var url = "http://db.sead.systems:8080/466419818?start_time=1446537600&end_time=1446624000&list_format=energy&type=P&device=Panel1&granularity=3600";
```
Comment (iangudger) [inline seadssite/static/js/graph.js]: Remove commented out code.

File: seadssite/static/js/graph.js [hunk 4]
```diff
@@ -77,53 +150,179 @@ function fetch_graph(url) {
...
+            format: 'MM/DD/YYYY HH:mm'
+        });
+
+        $('#start-date').data('DateTimePicker').setDate(start);
+        $('#end-date').data('DateTimePicker').setDate(end);
     });
 }
 
+function post_data_to_server(label) {
+    //POST
```
Comment (iangudger) [inline seadssite/static/js/graph.js]: Duplicate?

File: seadssite/static/js/timer.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    //console.log("" + start_time + " " + end_time);
+
+    $('#confirm-modal').modal('toggle');
+}
+
+function post_data_to_server(label) {
+    console.log("Sending " + JSON.stringify(label));
+
+    var post = new XMLHttpRequest();
+    var url = "http://db.sead.systems:8080/-1/label";
```
Comment (iangudger) [inline seadssite/static/js/timer.js]: Where is the device ID?

**Comment by ramhound** [comment]
Cleaned up the code to reflect the fact that we have strictly been implementing UI, and are currently working on server requests

**Comment by ianlofs** [comment]
this looks really awesome, just clean up the indentation stuff and ill merge

**Comment by ramhound** [comment]
I believe I fixed all indentation issues with last commit

**Pull request merged**